### DISCUSSION TITLE
Color in Contract seems to be incorrect in Dark mode

### DISCUSSION THREAD

**Discussion by deluxeGitHub** [kimai/kimai#5084] [General]
It seems the dark mode is broken after I Installed the 2.22 update (I was on 2.11 before)

![20240926163012](https://github.com/user-attachments/assets/6466c5a8-e507-4560-88a5-921af5be859e)

**Comment by kevinpapst** [comment]
I have seen these white boxes when working on 2.22.
The underlying theme was updated with 2.22 and it seems you didn't fully update your system.
The public assets (JS, CSS) are old.

Looks correct in the demo:
<img width="1268" alt="Bildschirmfoto 2024-09-26 um 16 50 14" src="https://github.com/user-attachments/assets/db5025c5-7828-469e-ab43-fc132f01ce8d">

**Reply by deluxeGitHub** [reply]
Hm...

I updated via docker-compose from Linusserver and even a fresh reinstall did not fix the issue.

I've switched back to Linuxserver.io version: 2.20.1-ls75 and that did fix the issue.
